src/composable/useFileLikeObject.js

```diff
@@ -0,0 +1,10 @@
+// method which work convert photo name in blob format
```

Comment:
method with description - bad method, work with names of variables, not comments

this function is OK, get rid of the comment please